Basic example at top of Readme (#214)
lucaslorentz/caddy-docker-proxy: README.md

Every time a docker object changes, it updates the Caddyfile and triggers a caddy zero-downtime reload.

> ## Basic Usage Example, using docker-compose:

Author: create network is necessary, so that 2 separate containers can talk to each other (and the www if external: true)

> ## Basic Usage Example, using docker-compose:

Reviewer: Not if they're both in the same file, and you don't make the network external... which is not necessary for a basic example. That's by definition a complication.

```
docker network create caddy
```
caddy/docker-compose.yml
```
version: "3.7"
services:
  caddy:
    image: lucaslorentz/caddy-docker-proxy:ci-alpine
    ports:
      - 80:80
      - 443:443
    networks:
      - caddy
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - caddy_data:/data
    restart: unless-stopped

networks:
  caddy:
    external: true
	
volumes:
  caddy_data: {}
```
```
docker-compose up -d
```
whoami/docker-compose.yml
```
version: '3.7'
services:
  whoami:
    image: jwilder/whoami
    networks:
      - caddy
    labels:
      caddy: whoami.example.com
      caddy.reverse_proxy: "{{upstreams 8000}}"

networks:
  caddy:
    external: true
```
```
docker-compose up -d
```

> visit whoami.example.com. The site with be served automatically over https with a Let's Encrypt Certificate

Reviewer: ```suggestion
Now you can visit `whoami.example.com`, which will be served with [Automatic HTTPS](https://caddyserver.com/docs/automatic-https).
```

Reviewer 2: In order to get it working it is probably required to have:
- chosen domain pointing to your machine
- port 80 is publicly accessible so the http1 challenge is solved

Should we mention something about this?
